lib/common/client.ts

```diff
@@ -5,28 +5,23 @@ import { registerStoryblokBridge } from "@storyblok/js";
 export const useStoryblokState: TUseStoryblokState = (
   initialStory = null,
   bridgeOptions = {},
-
 ) => {
-  let [story, setStory] = useState(initialStory);
+  const [story, setStory] = useState(initialStory);
 
-  const isBridgeEnable =
-    typeof window !== "undefined" &&
+  const storyId = (initialStory as any).internalId || initialStory.id;
```

Comment:
```suggestion
  const storyId = initialStory ? (initialStory as any).internalId || initialStory.id : 0;
```
As `initialStory` can be null

Reply:
We could also do something like this:
```suggestion
  const storyId = (initialStory as any)?.internalId ?? initialStory?.id;
```
I'm not 100% sure about the `0` else case though.

Reply:
> Just to be save I would suggest the following change:
> 
> As `initialStory` can be null

fixed